### PR TITLE
refactor: helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
Nice!

File: helm/templates/_resources.tpl [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{{/*
+Create service name as used by the service name label.
+*/}}
+{{- define "service.fullname" -}}
+{{- if not .Values.isExporter }}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "service" | indent 1 }}
+{{- else }}
+{{- printf "%s-%s-%s-%s" .Release.Name "exporter" .Chart.Name "service" | indent 1 }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:8]: fixed

File: helm/templates/_resources.tpl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+{{- end }}
+
+{{/*
+Create configmap name as used by the service name label.
+*/}}
+{{- define "configmap.fullname" -}}
+{{- if not .Values.isExporter }}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "configmap" | indent 1 }}
+{{- else }}
+{{- printf "%s-%s-%s-%s" .Release.Name "exporter" .Chart.Name "configmap" | indent 1 }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:19]: fixed

File: helm/templates/_resources.tpl [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+{{- end }}
+
+{{/*
+Create deployment name as used by the service name label.
+*/}}
+{{- define "deployment.fullname" -}}
+{{- if not .Values.isExporter }}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "deployment" | indent 1 }}
+{{- else }}
+{{- printf "%s-%s-%s-%s" .Release.Name "exporter" .Chart.Name "deployment" | indent 1 }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:30]: fixed

File: helm/templates/_resources.tpl [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+{{- end }}
+
+{{/*
+Create route name as used by the service name label.
+*/}}
+{{- define "route.fullname" -}}
+{{- if not .Values.isExporter }}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- else }}
+{{- printf "%s-%s-%s-%s" .Release.Name "exporter" .Chart.Name "route" | indent 1 }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:41]: fixed

File: helm/templates/_resources.tpl [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
+{{- if not .Values.isExporter }}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- else }}
+{{- printf "%s-%s-%s-%s" .Release.Name "exporter" .Chart.Name "ingress" | indent 1 }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl:52]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**